staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -506,6 +506,11 @@ type CreateOptions struct {
 	// +optional
 	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,1,rep,name=dryRun"`
 	// +k8s:deprecated=includeUninitialized,protobuf=2
+
+	// FieldManager is the name used to track resources for this
```

Comment:
A little confusing.  `fieldManager` to have correct case, then maybe something like:

> fieldManager is a name associated with the actor or entity that is making these changes. The allowed values are ...

and describe what characters are allowed.